common/src/main/java/org/keycloak/common/util/reflections/Reflections.java

```diff
@@ -988,7 +987,7 @@ public static boolean isPrimitive(Type type) {
      * @throws IllegalAccessException
      * @throws InstantiationException
      */
-    public static <T> T newInstance(final Class<T> fromClass) throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+    public static <T> T newInstance(final Class<T> fromClass) throws ClassNotFoundException, IllegalAccessException, InstantiationException, InvocationTargetException, NoSuchMethodException {
```

Comment:
Nitpick: Changing the signature of the method is also a breaking change at least when people want to compile their code. AFAIK this method is not used, so I'd suggest to either keep it as is for now, deprecate it for KC27 and eventually remove it.